packages/training/python/tests/test_evaluation.py

```
@@ -0,0 +1,727 @@
+- RolloutDumper for debugging/dataset generation
+- TestScenarioManager
+- BaselineManager
+"""
+
+import json
+import tempfile
+from datetime import datetime, timezone
+from pathlib import Path
+from typing import Dict, List
+
+import json
```

Ruff (0.14.10)

Redefinition of unused `json` from line 11

Remove definition: `json`

(F811)